docker/Makefile

```
@@ -0,0 +1,51 @@
+.PHONY: help build-all build-base build-languages build-vscode build-ai-tools test clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)